docs/_integration-manager/camel-routes/camel-route-local.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Camel Route Configuration for Local Storage
+nav_order: 1
+parent: Camel Route Configuration
+---
+# Camel Route Configuration for Local Storage
+
+## Overview
+
```

Comment:
## Overview

Monitor a local folder for new files and submit to the JobConfig run API

Reply:
Done.